### DISCUSSION TITLE
Question: Rectangle Able to Quickly Tile Bottom Right [see diagram]?

### DISCUSSION THREAD

**Discussion by dennisrcao** [rxhanson/Rectangle#1430] [General]
<img width="1332" alt="image" src="https://github.com/user-attachments/assets/77723574-0b04-416a-be7c-eddfb652cf28">
I use Rectangle all the time and love it, but was wondering if theres the ability to snap to a custom rectangle as I've shown in the diagram above? 

Lets say this diagram represents my TV (that I commonly use to code) and I regularly divide into sixths or thirds (using rectangle). I've looked through the app but can't seem to find the ability to create a shortcuts to tile a window into the **red rectangle** shown above.

**Comment by Aaron-Rumpler** [comment]
[Add additional "thirds" sizing commands](https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#add-additional-thirds-sizing-commands)
